src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems

```diff
@@ -9,23 +9,11 @@
     <Import_RootNamespace>AnalyzerDriver</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)AnalyzerDriverHelper.cs">
```

Comment:
I think there are a lot ExcludeFromStyleCop directives in a bunch of other project files. Can you clean those up too?